### PR TITLE
Gang War: Replaces sniper rifle with weaker version, nonlethal ammo, tweaks bling value

### PR DESCRIPTION
**Pull request by Robustin**
15 less damage, significantly less penetration, 1 tick stun instead of 5 tick stun, and 15 dismemberment down from 50. 

This should strike a balance where snipers are a good niche weapon but not something you can run around and 360 noscope with and expect to survive. If you're in good health and have your gang outfit on, you should survive 2 shots. 

Bling is also moderately cheaper, it was a smart investment before but prices were a tad high for most soldiers to consider. 

I've also added tranquilizer ammo to the sniper rifle. One thing I've found frustrating as a gang leader is that by the end-game you have to absolutely murder everyone you see. Even if they aren't dressed as a gangster, running in for a pen is often suicide. The tranquilizer ammo will deal 60 stamina damage and will sleep you if you're under 50 "health". So two consecutive shots will put you down unless you use a stimpack (which counters this pretty hard) or you can avoid being hit for ~5 seconds (sniper has a 4 second delay between shots) if you're in otherwise good health. This will give gangsters a non-lethal alternative that will let them convert rival gangsters and neutrals without having to resort to the sloppy crit + convert + heal combo.

### CODE REVIEW

**Comment by kevinz000** [comment]
damnit i was going to go "robustin added op weapons i guess i can do it too"
fugg XDDX

**Comment by Bawhoppen** [comment]
yea cause this will fix gang

**Comment by Robustin** [comment]
One step at a time. Having a mode without Stun2Win focused combat and deeper combat mechanics will be one of the hallmarks of New Gang War. The sniper rifle obviously cannot be a taser and a revolver in the same weapon.

**Comment by Kraseo** [comment]
how about instead of balancing influence around bling you balance it around # OTHER GANGSTERS ACTUALLY FUCKING TAGGING THE STATION

**Comment by Robustin** [comment]
It already is?

You're going to have to be more specific. Tagging is still the primary source of influence, bling is an interesting alternative mechanic that allows late-game gangsters to focus on eliminating rivals and dealing with dominators isntead of needing to feel glued to a spray can for 40 minutes.

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -279,6 +279,29 @@
...
+	stun = 0
+	weaken = 0
+	dismemberment = 0
+	breakthings = FALSE
+
+/obj/item/projectile/bullet/sniper/gang/sleeper/on_hit(atom/target, blocked = 0)
+	if((blocked != 100) && isliving(target))
+		var/mob/living/L = target
+		L.drowsyness += 6
+		if(L.health < 55)
```
Comment (kevinz000) [inline code/modules/projectiles/projectile/bullets.dm]: I'm 99% sure that health is mob health, that doesn't include stamina or I'd be able to crit people with a disabler.

Comment (Robustin) [inline code/modules/projectiles/projectile/bullets.dm]: I'll test it in a bit but I've definitely been crit by mixed stamina/health damage before.

**Comment by Robustin** [comment]
Tested, fixes made, ready to go

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**